### PR TITLE
C199656/fix

### PR DESCRIPTION
**Pull request by kimkate303**

### CODE REVIEW

File: cypress/integration/settings/circulation/patron-notices/create-patron-notice-policies.spec.js [hunk 1]
```diff
@@ -3,25 +3,27 @@ import NewNoticePolicy from '../../../../support/fragments/circulation/newNotice
...
-    NewNoticePolicy.choosePolicy(patronNoticePolicy);
-    NewNoticePolicy.duplicatePolicy(patronNoticePolicy);
-    NewNoticePolicy.deletePolicy(patronNoticePolicy);
-    NewNoticePolicy.choosePolicy(patronNoticePolicy);
-    NewNoticePolicy.editPolicy(patronNoticePolicy);
-    NewNoticePolicy.save(patronNoticePolicy);
-    NewNoticePolicy.check(patronNoticePolicy.name);
-    NewNoticePolicy.choosePolicy(patronNoticePolicy);
-    NewNoticePolicy.deletePolicy(patronNoticePolicy);
+    NewNoticePolicy.checInitialState();
```
Comment (MaksimTerekhov) [inline cypress/integration/settings/circulation/patron-notices/create-patron-notice-policies.spec.js:15]: typo in chec

File: cypress/integration/settings/circulation/patron-notices/create-patron-notice-template.spec.js [hunk 2]
```diff
@@ -11,9 +11,11 @@ describe('ui-circulation-settings: create patron notice template', () => {
 
   it('C199656 Create notice template', { tags: [TestType.smoke] }, () => {
     NewNoticePolicyTemplate.startAdding();
+    NewNoticePolicyTemplate.checInitialState();
```
Comment (MaksimTerekhov) [inline cypress/integration/settings/circulation/patron-notices/create-patron-notice-template.spec.js:14]: typo

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/integration/settings/circulation/patron-notices/create-patron-notice-policies.spec.js (hunk 1 above)
Comment (kimkate303) [reply cypress/integration/settings/circulation/patron-notices/create-patron-notice-policies.spec.js:15]: omg

File: cypress/integration/settings/circulation/patron-notices/create-patron-notice-policies.spec.js [hunk 3]
```diff
@@ -3,25 +3,27 @@ import NewNoticePolicy from '../../../../support/fragments/circulation/newNotice
...
 describe('ui-circulation-settings: create patron notice policies', () => {
-  const patronNoticePolicy = { ...NewNoticePolicy.defaultUiPatronNoticePolicies };
+  const noticePolicy = { ...NewNoticePolicy.defaultUi };
   beforeEach('login', () => {
     cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
     cy.visit(`${SettingsMenu.circulationPatronNoticePoliciesPath}`);
   });
 
   it('C6530 Create notice policy', { tags: [TestType.smoke] }, () => {
+    NewNoticePolicy.waitLoading();
```
[the remark this answers is not in the document]
Comment (kimkate303) [reply cypress/integration/settings/circulation/patron-notices/create-patron-notice-policies.spec.js:13]: yep fixed it https://github.com/folio-org/stripes-testing/commit/51dbdbeb769f4f3e7a860d0ac354879d15851ddb

**Comment by kimkate303** [comment]
@dmitriyfolio please check, added loginAsAdmin

**Review by dmitriyfolio — APPROVED** [review]

**Pull request merged**